tools/contractdeployer/contract_deployer.go

```diff
@@ -23,7 +23,8 @@ import (
 // The types of contracts supported by the deployer
 const (
 	mgmtContract         = "MGMT"
-	erc20Contract        = "ERC20"
+	obsErc20Contract     = "OBSERC20"
+	ethErc20Contract     = "ETHERC20"
```

Comment:
I might be tempted to call these L1 and L2 ERC20 contracts. Just because there will be an ETH version of both of them for example

Reply:
yeah, I actually had the same thought when I looked at it afterwards.
Will rename